tips/TIP-0018/tip-0018.md

```diff
@@ -0,0 +1,3755 @@
+validation on its past cone. The timestamp and milestone index of the confirming milestone provide the time as an input
+parameter to transaction validation.
+
+An output that contains a <i>Timelock Unlock Condition</i> can not be unlocked before the specified timelock has
+expired. The timelock is expired when the timestamp and/or milestone index of the confirming milestone is equal or past
+the timestamp and/or milestone defined in the <i>Timelock Unlock Condition</i>.
+
+The timelock can be specified as a unix timestamp or as a milestone index. When specified in both ways, both conditions
+have to pass in order for the unlock to be valid. The zero value of one if the timestamp fields signals that it should be
+ignored during validation.
+
+The two time representations help to protect against the possible downtime of the Coordinator. If the Coordinator is
+down, "milestone index clock" essentially stops advancing, while "real time clock" does not. An output that specifies
+time in both clocks must satisfy both conditions (AND relation).
+
+###### Additional syntactic transaction validation rules:
+- If both `Milestone Index` and `Unix Time` fields are `0`, the unlock condition, and hence the output and transaction
+  that contain it, is invalid.
+###### Additional semantic transaction validation rules:
+- An output that has <i>Timelock Unlock Condition</i> specified must only be consumed and unlocked in a
+  transaction, if the confirming milestone index is ≥ than the `Milestone Index` specified in the unlock condition.
+- An output that has <i>Timelock Unlock Condition</i> specified must only be consumed and unlocked in a
+  transaction, if the timestamp of the confirming milestone is equal or past the `Unix Time` specified in the unlock
+  condition.
+
+<details>
+<summary>Timelock Unlock Condition</summary>
+<blockquote>
+    Defines a milestone index and/or unix timestamp until which the output can not be unlocked.
+</blockquote>
+</details>
+
+<table>
+    <tr>
+        <td><b>Name</b></td>
+        <td><b>Type</b></td>
+        <td><b>Description</b></td>
+    </tr>
+    <tr>
+        <td>Unlock Condition Type</td>
+        <td>uint8</td>
+        <td>
+            Set to <strong>value 2</strong> to denote a <i>Timelock Milestone Index Unlock Condition</i>.
+        </td>
+    </tr>
+    <tr>
+        <td>Milestone Index</td>
+        <td>uint32</td>
+        <td>
+            The milestone index starting from which the output can be consumed.
+        </td>
+    </tr>
+    <tr>
+        <td>Unix Time</td>
+        <td>uint32</td>
+        <td>
+            Unix time (seconds since Unix epoch) starting from which the output can be consumed.
+        </td>
+    </tr>
+</table>
+
+##### Expiration Unlock Conditions
+
+The expiration feature of outputs makes it possible for the sender to reclaim an output after a given expiration time
```

Comment:
Should be return address instead of sender ?
Same for next paragraph.

Reply:
In the next paragraph, sender is used in other context (sender-receiver) not as the name of the field.